src/components/helper/Text/index.jsx

```diff
@@ -0,0 +1,88 @@
+//** Standard Frameworks */
+import React from "react";
+
+class Text extends React.Component{
+    getItem = (segment) => {
+        //** Regex cancer */
```

Comment:
NOTE: This is not a accurate comment of the functionality of inner workings of the described regex.